src/ScheduleFunctions.cpp

```diff
@@ -711,7 +711,7 @@ Stmt inject_explicit_bounds(Stmt body, Function func) {
 }
 
 class IsUsedInStmt : public IRVisitor {
-    string func;
+    const string& func;
```

Comment:
micronit: space before &

Reply:
Okay, added. I'll push that when I figure out what's causing build failures.